ui/fetch_ui_assets.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+curl -L https://github.com/influxdata/ui/releases/download/OSS-Master/build.tar.gz --output build.tar.gz
```

Comment:
Can you add a comment above this line explaining how/when `OSS-Master/build.tar.gz` is generated? i.e. does it get updated on every merge to the `OSS-Master` branch in the `ui` repo?

Reply:
I added some comments to the script!

At this point the asset generation from `ui` repo `master` is a manual process. When the `monitor-ci` testing infrastructure gets set up with OSS, I will update the build process to re-generate the `build.tar.gz` & update the `OSS-Master` release. We can't do that just yet though because the e2e tests that still run in `influxdb` might break. The automated testing with `monitor-ci` will take care of that since it will also include testing UI changes against the OSS backend and we will be able to stop running the e2e in the OSS repo (altough e2e tests will still run for each build - they'll just come from `monitor-ci`).

Feature release branches of OSS will work differently - infrequent updates, only for bug fixes in the UI specific to that release. Implicit in this is that the `fetch_ui_assets.sh` script will need to point to a different URL in a release branch, like `https://github.com/influxdata/ui/releases/download/OSS-2.0/build.tar.gz`. Perhaps this too could be automated somehow in the future.